exchanges/deribit/deribit_websocket.go

```diff
@@ -496,22 +502,26 @@ func (d *Deribit) processTrades(respRaw []byte, channels []string) error {
 		if err != nil {
 			return err
 		}
-		side, err := order.StringToOrderSide(tradeList[x].Direction)
-		if err != nil {
-			return err
-		}
 		tradeDatas[x] = trade.Data{
 			CurrencyPair: cp,
 			Exchange:     d.Name,
-			Timestamp:    tradeList[x].Timestamp.Time(),
+			Timestamp:    tradeList[x].Timestamp.Time().UTC(),
 			Price:        tradeList[x].Price,
 			Amount:       tradeList[x].Amount,
-			Side:         side,
+			Side:         tradeList[x].Direction,
 			TID:          tradeList[x].TradeID,
 			AssetType:    a,
 		}
 	}
-	return trade.AddTradesToBuffer(tradeDatas...)
+	if tradeFeed {
+		for i := range tradeDatas {
```

Comment:
🥃  Data is already plural. Better to treat it like a postpositive adjective in this situation, like `courts martial`, `surgeons general`.
So `tradesData`.
Also tradesDataset is viable; Datum => Data => Dataset.

Reply:
@Beadko